docs/success-stories/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: success-stories
+show_sidebar: false
+permalink: /success-stories/
+
+# Banner Content
+# =================================
+title: Success Stories
+subtitle: Read about some of the amazing outcomes that are being realized by our platform.
+hero_image: /assets/img/home-callouts/success_stories.jpg
+
+# Stories
+# =================================
+stories:
+ - story-id: 1
+   title: The Digital Slide Archive and TCGA
+   description-short: "The Cancer Digital Slide Archive is a website that hosts whole-slide images of pathologic specimens associated with the TCGA studies ..."
+   page-link: success-story-The-Digital-Slide-Archive-and-TCGA
+   image: assets/img/success-stories/story-image-1.png
+ - story-id: 2
+   title: Developing Machine-Mearning Tools Through Collaborative Annotation Studies
```

Comment:
Machine-Learning

Reply:
fixed in 36492a6a3950eef0308c2102ac04fe1ae1f3df98